gulpfile.js

```diff
@@ -322,8 +385,9 @@
                 .watch(
                     [
                         '*.js',
-                        'public/app/**',
-                        'test/**/*.js'
+                        'public/app/**/*.js',
+                        'test/fixtures/**/*.js',
```

Comment:
When `test/main.js` was being regenerated, it was causing the `watch` task to activate, which regenerated `test/main.js`, which activated the `watch` task...  So it's no longer looking at `test/main.js`, although it is still linted.